config/routes.rb

```diff
@@ -3,6 +3,9 @@
   namespace :hq do
     resources :partners, except: [:destroy] do
       resources :orphan_lists, only: [:index]
+      resources :pending_orphan_lists, only: [:create, :new, :destroy] do
+        ['upload', 'validate', 'import'].each { |action| get action, on: :collection }
```

Comment:
So, this generates 6 routes for `pending_orphan_lists` in total? Is `:create` used?